backend/logistica_servicio/generador_rutas_entrega/optimizador_rutas/algoritmo/algoritmo_calculo_ruta.py

```
@@ -0,0 +1,64 @@
+
+    for i, point1 in enumerate(all_points):
+        graph[tuple(point1)] = {}
+        for j, point2 in enumerate(all_points):
+            if i != j:
+                graph[tuple(point1)][tuple(point2)] = haversine(point1, point2)
+
+    return graph
+
+def calcular_mejor_ruta(punto_inicio, destinos):
+    graph = create_graph(punto_inicio, destinos)
+    all_points = [punto_inicio] + destinos
```

Ruff (0.8.2)

Local variable `all_points` is assigned to but never used

Remove assignment to unused variable `all_points`

(F841)